e2e/rbd.go

```diff
@@ -290,7 +297,24 @@ var _ = Describe("RBD", func() {
 			Skip("Skipping RBD E2E")
 		}
 		c = f.ClientSet
-		if deployRBD {
+		if operatorDeployment {
+			rbdDeploymentName = operatorRBDDeploymentName
+			rbdDaemonsetName = operatorRBDDaemonsetName
+
+			err := setDomainLabels([]string{nodeRegionLabel, nodeZoneLabel})
```

Comment:
Steps like this should probably be part of a `DeploymentMethod` struct or interface. All the `if operatorDeployment` and `if !helmTest` make it very confusing to follow this code while reading it. Possibly things like this can be based on the `ResourceDeployer` interface, and a list of those structs can be executed depending on the `DeploymentMethod`.